src/js/containers/search/filters/psc/PSCListContainer.jsx

```diff
@@ -0,0 +1,168 @@
+        }
+
+        this.setState({
+            autocompletePSC: values
+        });
+    }
+
+    queryAutocompletePSC(input) {
+        this.setState({
+            noResults: false
+        });
+
+        // Only search if input is 2 or more characters
+        if (input.length >= 2) {
+            this.setState({
+                pscSearchString: input
+            });
+
+            if (this.pscSearchRequest) {
+                // A request is currently in-flight, cancel it
+                this.pscSearchRequest.cancel();
+            }
+
+            const pscSearchParams = {
+                search_text: this.state.pscSearchString
+            };
+
+            this.pscSearchRequest = SearchHelper.fetchPSC(pscSearchParams);
+
+            this.pscSearchRequest.promise
+                .then((res) => {
+                    let autocompleteData = [];
+                    const data = res.data.results;
+
+                    const selectedPSC =
+                        this.props.selectedPSC.toArray().map((psc) => omit(psc, 'identifier'));
+                    // Filter out any selectedPSC that may be in the result set
+                    if (selectedPSC && selectedPSC.length > 0) {
+                        autocompleteData = differenceWith(data, selectedPSC, isEqual);
+                    }
+                    else {
+                        autocompleteData = data;
+                    }
+
+                    this.setState({
+                        noResults: data.length === 0
+                    });
+
+                    // Add search results to Redux
+                    this.props.setAutocompletePSC(autocompleteData);
+                })
+                .catch((err) => {
+                    if (!isCancel(err)) {
+                        this.setState({
+                            noResults: true
+                        });
+                    }
+                });
+        }
+        else if (this.pscSearchRequest) {
+            // A request is currently in-flight, cancel it
+            this.pscSearchRequest.cancel();
+        }
+    }
+
+    clearAutocompleteSuggestions() {
+        this.props.setAutocompletePSC([]);
+    }
+
+    handleTextInput(pscInput) {
+        // Clear existing PSC to ensure user can't select an old or existing one
+        this.props.setAutocompletePSC([]);
+
+        // Grab input, clear any exiting timeout
+        const input = pscInput.target.value;
+        window.clearTimeout(this.timeout);
+
+        // Perform search if user doesn't type again for 300ms
+        this.timeout = window.setTimeout(() => {
+            this.queryAutocompletePSC(input);
+        }, 300);
+    }
+
+    render() {
+        return (
+            <Autocomplete
+                {...this.props}
+                values={this.state.autocompletePSC}
+                handleTextInput={this.handleTextInput}
+                onSelect={this.props.selectPSC}
+                placeholder="eg: 1510 - Aircraft, Fixed Wing"
```

Comment:
Nitpick: can we change this to `e.g.,` like in Budget Categories _or_ change Budget Categories to match this `eg:`.